changes/4788.fix.md

```diff
@@ -0,0 +1 @@
+Add missing status code in wsproxy exception middleware
```

Comment:
Could you rewrite the sentence to express the fact that status code has not been returned occasionally (when accept header does not equal to `application/json`)?